.github/workflows/publish_to_psgallery.yml

```diff
@@ -0,0 +1,34 @@
+name: publish_to_psgallery
+on:
+- push
+- workflow_dispatch
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    name: Publish to PowerShell Gallery
+    # only trigger for pushed tags
+    if: |
+      startsWith(github.event.ref, 'refs/tags/')
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get version from tag ref
+      id: get_version
+      run: |
+        regex="refs\/tags\/"
+        echo "::set-output name=VERSION::${GITREF/$regex/}"
+      env:
+        GITREF: ${{ github.event.ref }}
+
+    - run: |
+        Update-ModuleManifest -Path .\module\Corvus.Deployment.psd1 `
```

Comment:
Does this have to be hard coded? Could this be passed in as a variable or something? Otherwise it looks like this could be completely re-usable?

Reply:
I've updated it to use a separate configuration file which allows the workflow to be generic, whilst keeping the config in source and avoiding the need to setup github secrets.